docs/data/tree-view/rich-tree-view/editing/editing.md

```diff
@@ -0,0 +1,86 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Editing
+githubLabel: 'component: tree view'
+packageName: '@mui/x-tree-view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+packageName: '@mui/x-tree-view'
+---
+
+# Rich Tree View - label editing
+
+<p class="description">Learn how to edit the label of Tree View items.</p>
+
+:::warning
```

Comment:
I think you can drop this comment now that we are close from the open review
Tbh I would even open for review today to gather feedbacks